core/opendaq/reader/include/opendaq/block_reader.h

```diff
@@ -83,13 +83,20 @@ DECLARE_OPENDAQ_INTERFACE(IBlockReader, ISampleReader)
      * @param[out] size The number of samples in a block.
      */
     virtual ErrCode INTERFACE_FUNC getBlockSize(SizeT* size) = 0;
+
+    /*!
+     * @brief The amount of block overlapping.
+     * @param[out] overlap The overlap size in percents.
+     */
+    virtual ErrCode INTERFACE_FUNC getOverlap(SizeT* overlap) = 0;
 };
 /*!@}*/
 
 OPENDAQ_DECLARE_CLASS_FACTORY(
     LIBRARY_FACTORY, BlockReader,
     ISignal*, signal,
     SizeT, blockSize,
+    SizeT, overlap,
```

Comment:
These factory changes, as well as the interface modifications require a python binding update. I think the block reader is not automatically generated, so the parameters need to be added manually.

Reply:
Reverted